presto-native-execution/presto_cpp/main/PrestoExchangeSource.cpp

```diff
@@ -132,7 +132,7 @@ bool PrestoExchangeSource::shouldRequestLocked() {
 
 folly::SemiFuture<PrestoExchangeSource::Response> PrestoExchangeSource::request(
     uint32_t maxBytes,
-    uint32_t maxWaitSeconds) {
+    std::chrono::microseconds maxWait) {
```

Comment:
Nit : Rename variable to maxWaitMcs (seems like other time variables in this code have shorthand for their unit in the name).

Reply:
We had the unit because we were using integer, now we encode the unit in the type itself so it is no longer needed